atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -64,6 +68,47 @@ def override_build_kwarg(workflow, k, v):
     override_kwargs[k] = v
 
 
+class UnknownPlatformException(Exception):
+    """ No clusters could be found for a platform """
+
+
+class AllClustersFailedException(Exception):
+    """ Each cluster has reached max_cluster_fails """
+
+
+class ClusterRetryContexts(object):
+    def __init__(self, clusters, max_cluster_fails, find_cluster_retry_delay):
+        self.clusters = set(clusters)
```

Comment:
I originally put a self.clusters here thinking it would be useful to store the cluster info inside this class. If we're not using it that way though, let's just remove that attribute.

Reply:
Oh, I see, it's used in `wait_for_any()` just to see if we've run out of clusters to try.

I wonder if, instead, we should change the caller of this method (i.e. `get_clusters`) so it raises an exception if it knows there are no clusters left to try -- it already knows this, without having to calculate it.